src/Controller/Api/TwoFactorAuthController.php

```diff
@@ -11,14 +11,14 @@
 use Nelmio\ApiDocBundle\Annotation\Model;
 use Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Google\GoogleAuthenticatorInterface;
 use Swagger\Annotations as SWG;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
-final class TwoFactorAuthController extends Controller
+class TwoFactorAuthController extends AbstractController
```

Comment:
Best practice setting the final to the class if this class will not be extending